packages/node_modules/@webex/redux-module-spaces/src/actions.js

```diff
@@ -124,7 +124,7 @@ function addLoadError(dispatch, code) {
     code,
     id: 'redux-module-spaces-load',
     displayTitle: 'Something\'s not right',
-    displaySubtitle: 'Unable to load spaces. Please try again later.',
+    displaySubtitle: `Unable to load spaces. Please try again later. [${code}]`,
```

Comment:
Is this adding the error code from the HTTP call?

Reply:
This adds the mutated code from the HTTP call, the JS SDK converts the code into a string, and it is provided in this method as the `code` parameter.

Reply:
It looks like all the calls to this function are doing:
```js
.catch((err) => {
        addLoadError(dispatch, err.name);

        throw (err);
      });
```

We should change the param name from "code" to "name"? Not really in scope of this ticket, but could be a quick win.

Reply:
The name can also exist as a code if it isn't an error code that is mapped by the JS SDK. So, if we receive something like `999`, it will be `999` in the brackets, as it doesn't map to a known name in the SDK, see the status code interceptor for more details.

Reply:
yeah, calling it `code` is misleading

Reply:
update this snippet.